website/data-storage/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:16-alpine
```

Comment:
There is good practice to specify docker image with hash.

> Be very specific! Instead of pulling a tag, pull an image using the specific SHA256 reference of the Docker image, which guarantees you get the same image for every pull. However notice that using a SHA256 reference can be risky, if the image changes that hash might not exist anymore.
https://snyk.io/blog/10-docker-image-security-best-practices/